docs/reference/settings/security-settings.asciidoc

```diff
@@ -1770,6 +1770,7 @@ As per `claim_patterns.principal`, but for the _dn_ property.
 (<<static-cluster-setting,Static>>)
 The maximum allowed clock skew to be taken into consideration when validating
 id tokens with regards to their creation and expiration times.
+Defaults to `60m`.
```

Comment:
I believe this is 60 seconds instead of 60 minutes:
```suggestion
Defaults to `60s`.
```

Reply:
Thank you for catching my typo!